tgui/packages/tgui/styles/interfaces/WirePanel.scss

```diff
@@ -0,0 +1,169 @@
+@use '../colors.scss';
+@use '../components/Button.scss';
+@use '../base.scss';
+
+.WirePanel-wires-container {
```

Comment:
Broadly have quite a few comments for this file, certainly in how it's structured. As some stuff is still in flight I don't want to give too detailed a review here just yet. Please reach out to me when you think the functionality is more stable and thus we can work on tidying up the SCSS.